src/right-field/RightField.jsx

```diff
@@ -186,15 +206,23 @@ function disableIterator() {
   }
 }
 
+async function fetchQuery(query, eventEmitter) {
+  try {
+    let result = await fetch(`${config.queryFolder}${query.queryLocation}`);
```

Comment:
Why is `let` used here?

Reply:
There is no reason for this and I have changed it in [commit 091de4e](https://github.com/SolidLabResearch/generic-data-viewer/pull/24/commits/091de4e880929b4a8c27cfa3a77a75b012576ad4)